api_methods.py

```diff
@@ -0,0 +1,132 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def jsonify_response(url):
+    """Send get request and return data in JSON."""
+    response = requests.get(url)
+    if response.status_code == 200:
+        return json.loads(response.text)
+
+
+def get_categories():
```

Comment:
Done.
Not sure should I initialize an instance with search parameters, or should I pass them into `.get_average()` as in was before. Please advice.

Also I have no idea what should be described in class docstring (just copy-pasted there module docstring), and the same for class `__init()__` method.